src/xunit.analyzers/AssertCollectionEmptyCheckShouldNotUseCollectionCheckFixer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace Xunit.Analyzers
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public class AssertCollectionEmptyCheckShouldNotUseCollectionCheckFixer : CodeFixProvider
+    {
+        private const string Title = "Use Assert.Empty";
+
+        public sealed override ImmutableArray<string> FixableDiagnosticIds { get; } = ImmutableArray.Create(Descriptors.X2011_AssertCollectionEmptyCheckShouldNotUseCollectionCheckFixer.Id);
+
+        public sealed override FixAllProvider GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
+
+        public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+            var invocation = root.FindNode(context.Span).FirstAncestorOrSelf<InvocationExpressionSyntax>();
+            
+            context.RegisterCodeFix(
```

Comment:
I think there should be an alternative fixer for this, which would add an element inspector. If you don't have the bandwidth/interest to implement that we could open a follow up issue instead.